File: Practice(Beginner)/C++/Chef and Remissness.cpp
// Source: https://www.codechef.com/problems/REMISS

#include <bits/stdc++.h>
using namespace std;

int main() {

    int t, a, b;
    cin >> t;
    while (t--){
        cin >> a >> b;
        cout << max(a,b) << " " << a+b << endl;
    }

    return 0;
}

File: Practice(Beginner)/Python3/Chef_and_Remissness.py
# Source: https://www.codechef.com/problems/REMISS

n = int(input())
for i in range(n):
    a,b = map(int,input().split())
    if(a > b):
        min_enter = a
    else:
        min_enter = b
    max_enter = a+b
    print(min_enter,max_enter)
